models/shopify__daily_shop.sql

```diff
@@ -0,0 +1,213 @@
+with shop as (
+
+    select *
+    from {{ var('shopify_shop') }}
+),
+
+calendar as (
+
+    select *
+    from {{ ref('shopify__calendar') }}
+    where cast({{ dbt.date_trunc('day','date_day') }} as date) = date_day
+
+),
+
+shop_calendar as (
+
+    select
+        calendar.date_day,
+        shop.*
```

Comment:
Do we really want to include all these fields in the end model? The heavy metrics work great, but some of the dimensional fields seem to add a lot of unnecessary noise. For example, do we need to know `is_google_apps_login_enabled` for each day of the shop? Additionally, I don't think we have that historical info so it will always be either `true` or `false` in this scenario.

Can we limit the shop fields brought through to only those that will help identify the shop. ANything else may be joined in by the customer if they wish.

Reply:
limited them -- lemme know whatcha think

Reply:
That looks great, thanks for the update!